app/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -9,7 +9,7 @@ function PopupWithForm(props) {
         ></button>
         <h2 className="popup__title">{props.title}</h2>
         {props.children ? (
-          <form className="popup__form" action="submit" noValidate>
+          <form className="popup__form" action="submit">
```

Comment:
Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses